### ISSUE TITLE
External chapter example

### ISSUE THREAD

**Issue by mahemoff** [podlove/podlove-specifications#18]
It would be useful to see an example of an external chapter resource at the bottom of the [Simple Chapters spec](https://github.com/podlove/podlove-specifications/blob/master/podlove-simple-chapters.md). It says this is a resource  "containing the chapter information", but would be safer for both publishers and developers if it included a complete example.

**Comment by funkenstrahlen** [comment]
I also do not quite understand how this is intended to work.

**Comment by Cj-Malone** [comment]
I've always assumed you'd just link to a file like this.

```
<?xml version="1.0" encoding="utf-8"?>
<chapters version="1.1" xmlns="http://podlove.org/simple-chapters">
  <chapter start="0" title="Welcome" />
  <chapter start="3:07" title="Introducing Podlove" href="http://podlove.org/" />
  <chapter start="8:26.250" title="Podlove WordPress Plugin" href="http://podlove.org/podlove-podcast-publisher" />
  <chapter start="12:42" title="Resumée" />
</chapters>
```

**Comment by funkenstrahlen** [comment]
Is `<?xml version="1.0" encoding="utf-8"?>` required?

**Comment by funkenstrahlen** [comment]
Also: What is the correct filename extension?

`filename.psc.txt` or `filename.psc`?

**Comment by timpritlove** [comment]
The XML header is required in any XML file. The filename does not matter in general, `.psc` is our convention.

**Comment by mahemoff** [comment]
@timpritlove Is @Cj-Malone's comment correct then? It's just a xml "chapters" file?

**Comment by timpritlove** [comment]
Yes, that's what is intended. You either place Podlove Simple Chapters inline or you link to a file containing an XML document adhering to the same namespace.

**Comment by johnloringpollard** [comment]
Are there any feeds doing external file chapters and do players support it?
